src/app/features/test-records/views/test-record/test-record.component.spec.ts

```diff
@@ -106,92 +110,6 @@ describe('TestRecordComponent', () => {
       el.query(By.css('button#save-test-result')).triggerEventHandler('click', {});
       expect(component.handleSave).toHaveBeenCalledTimes(1);
     });
-
```

Comment:
Cancel button is no more so test are not applicable anymore while the `isTestTypeEditable` stuff has moved elsewhere along with tests